app/models/loan.rb

```diff
@@ -3,4 +3,19 @@ class Loan < ActiveRecord::Base
   belongs_to :order
   validates :order_id, presence: true
   validates :project_id, presence: true
+  validates :amount, presence: true, numericality:
+    {
+     greater_than_or_equal_to: 1000,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.